### ISSUE TITLE
Scan does not work when the ComicInfo.xml tag <LocalizedSeries> is the same value as the .cbz name

### ISSUE THREAD

**Issue by Two-Pato** [Kareadita/Kavita#3400]
### What happened?

If the `<LocalizedSeries>` tag in the `ComicInfo.xml` file (e.g., `<LocalizedSeries>Test</LocalizedSeries>`) matches the name of the `.cbz` file (e.g., `Test v01.cbz`), Kavita will ignore the file during scanning. Changing either the file name or the tag value will allow it to scan correctly.

### What did you expect?

The scan should add all files.

### Kavita Version Number - If you don not see your version number listed, please update Kavita and see if your issue still persists.

0.8.4 - Stable

### What operating system is Kavita being hosted from?

None

### If the issue is being seen on Desktop, what OS are you running where you see the issue?

None

### If the issue is being seen on Mobile, what OS are you running where you see the issue?

None

**Comment by majora2007** [comment]
When a file has the series as Test and another file has the LocalizedSeries as Test, they will merge together into one series. This is intentional behavior. 

I'm not understanding what you are reporting as an issue.

**Comment by Two-Pato** [comment]
Yes you are actually right. If you have more than one .cbz file like Test v01.cbz and Test v02.cbz and so on, than it scans normally. But with only one .cbz file it does not. If you cant replicate this, its maybe something on my end.

**Comment by majora2007** [comment]
I'm not understanding what you mean then. If only one file, the file doesn't ingest when localized series is set? What do your logs say?

**Comment by Two-Pato** [comment]
Let's say I have a Japanese one-shot manga. I'll call it One Piece. In my library folder (where Kavita scans), I have now a folder named "One Piece," and within this folder is only one file "One Piece v01.cbz." In my ComicInfo.xml file, I have the tags:

<Title>One Piece</Title>
<'LocalizedSeries>One Piece<'/LocalizedSeries>

Since the title is the same in both Japan and the USA. When I press "Scan Library," the file does not show up in my library.

If I add the file "One Piece v02.cbz," it scans normally. The one-shot is now displayed with 2 volumes.

If I rename the file from "One Piece v01.cbz" to "Two Piece v01.cbz," it scans normally. The one-shot will show up. Its name is Two Piece the localized name One Piece.

If I delete or change the <'LocalizedSeries>One Piece<'/LocalizedSeries> tag, it scans normally. The one-shot will show up.

But if both the name of the .cbz file and the <'LocalizedSeries> tag are the same, and only one file exists, it does not appear in my library. If this is bad tagging from me I will just change this. But in previous versions, kavita did not care and just added everything.

**Comment by majora2007** [comment]
Okay this sounds like an edge case. This would be considered an edge case where the Series and LocalizedSeries are the same for a single file. But this is not an invalid use case, so I will look into it.
